### ISSUE TITLE
Helix Ultimate blog image selection

### ISSUE THREAD

**Issue by toonetcreation** [Quantum-Manager/tracker#84]
Hi,

I'm using Joomshaper Helix Ultimate template/framework.

Today, when I select a featured image from BLOG OPTIONS tab (see here : https://prnt.sc/1g2k8mc), I cannot choose an image from joomla media manager.

The system open a window from local computer.

Is it possible to redirect this button to quantum media manager ?

Thanks
L.

**Comment by dmitriitux** [comment]
hi, 
For some reason I cannot open the site with the screenshot. Attach to github in a message here. Drag-grop to the text field and it will attach

**Comment by toonetcreation** [comment]
here is as attached.

![image](https://user-images.githubusercontent.com/20813754/127019941-a98d8ddc-71b5-4dc2-adb0-98fa4419ae89.png)

**Comment by dmitriitux** [comment]
I have not worked with this template, so I ask.

where is the BLOG OPTIONS tab? in com_content? how can I repeat it. to put a template and start creating material in com_content?

**Comment by toonetcreation** [comment]
I spoke with the developer an he told me this :

```
plugins/system/helixultimate/html/layouts/form/field/media.php

templates/shaper_helixultimate/html/layouts/libraries/cms/html/bootstrap/starttabset.php

templates/shaper_helixultimate/html/com_content/form/edit.php

plugins/system/helixultimate/assets/js/admin/blog-options.js

plugins/system/helixultimate/fields/heliximage.php
```

**Comment by toonetcreation** [comment]
I can send to you the template package if you need it?

**Comment by dmitriitux** [comment]
yes, if you can, please send. email: cymbal@delo-design.ru

**Comment by toonetcreation** [comment]
in fact you can download it for free from here : https://www.joomshaper.com/downloads/template/helixultimate

;-)

**Comment by dmitriitux** [comment]
@toonetcreation ok

**Comment by toonetcreation** [comment]
thank you ;-)
do you think this improvement could be possible?

**Comment by dmitriitux** [comment]
@toonetcreation yes, it is possible. I am now accumulating and making a plan of what I need to do for the manager. It is necessary to implement a new image editor, issue a warning about file replacement, make group renames, adapt to several extensions, add hotkey schemes. And soon we need to start transferring to joomla 4. I'm just making a plan now and will gradually implement it. It's just that this task is not important for me now. =)

**Comment by dmitriitux** [comment]
I'll probably draw up a plan somewhere and post on the github what I'm doing according to the manager

**Comment by dmitriitux** [comment]
there are just a number of more important tasks, I will draw up a plan and throw off the link so that you understand that I

**Comment by toonetcreation** [comment]
I understand perfectly.
thank you for the feedback, very appreciated.
L.

**Comment by dmitriitux** [comment]
https://github.com/Quantum-Manager/tracker/issues/87

**Comment by toonetcreation** [comment]
for info : new major release of helix template delivered here : https://www.joomshaper.com/helix

**Comment by dmitriitux** [comment]
saw, and they have dynamic content?

**Comment by toonetcreation** [comment]
I don't understand your question sorry.
what do you mean exactly?

**Comment by dmitriitux** [comment]
example: https://yootheme.com/support/yootheme-pro/joomla/dynamic-content
we only use yootheme pro if a similar solution is required on builders due to dynamic content

**Comment by dmitriitux** [comment]
and, this is probably more necessary for the SP Builder, do not use this builder? is there an analogue of dynamic content?

**Comment by dmitriitux** [comment]
I'm just confused already, helix is a template, and the SP builder is a builder that runs on the helix template?

**Comment by toonetcreation** [comment]
Helix is the template/framework
SP Page Builder is a page Builder.
They Can work separately.
Blog options are provided by Helix, not by SP page Builder.

**Comment by dmitriitux** [comment]
understood

**Comment by toonetcreation** [comment]
😉😀
